api/catalog/api/views/media_views.py

```diff
@@ -211,6 +212,19 @@ def _thumbnail_proxy_comm(
             )
 
             return upstream_response, res_status, content_type
+        except requests.ReadTimeout as exc:
+            # Count the incident so that we can identify providers with most timeouts.
+            domain = urlparse(params["url"]).netloc
+            key = f"{settings.THUMBNAIL_TIMEOUT_PREFIX}__{domain}"
```

Comment:
We currently use `:` to separate prefixes rather than `__`. What is the reason for the difference here?

https://github.com/WordPress/openverse-api/blob/9f8b15773df511b0900e2a0adf7785382abee450/api/catalog/api/utils/validate_images.py#L16

Reply:
I didn't remember we used `:` prefix. Thanks for pointing that out. Fixed in 3b9eca4.